README.rst

```diff
@@ -26,13 +27,13 @@ Run integration tests against stage deployment:
 
 .. code-block:: console
 
-  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja ./test.sh
+  THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja THOTH_AMUN_API_HOST=amun.stage.thoth-station.ninja ./test.sh
 
 Run integration tests against test deployment (default behaviour):
 
 .. code-block:: console
 
-  THOTH_USER_API_HOST=test.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.test.thoth-station.ninja ./test.sh
+  THOTH_USER_API_HOST=test.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.test.thoth-station.ninja THOTH_AMUN_API_HOST=amun.test.thoth-station.ninja ./test.sh
```

Comment:
Can we make the endpoint `amun-api-thoth-test-core.apps.ocp.prod.psi.redhat.com` as `amun.test.thoth-station.ninja`?

Reply:
https://console-openshift-console.apps.ocp.prod.psi.redhat.com/k8s/ns/thoth-test-core/routes
we need to direct route53 to ocp cluster